miqa/core/tests/pyppeteer/test_import_export.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+
+
+async def get_current_num_scans(page):
+    return len(await page.xpath('//ul[contains(@class, "scans")]/li'))
+
+
+async def edit_import_export_paths(page):
+    # Edit import path
+    import_path_input = await page.waitForXPath('//input[contains(@name, "import-path")]')
+    await import_path_input.click({'clickCount': 3})  # highlight the text to overwrite it
+    await import_path_input.type('samples/demo_project.csv')
```

Comment:
I think it's worth having `import_path` and `export_path` arguments to this function. If someone were reading the test, they would need to dig into this method to find what exactly is being put into the import/export fields.